src/System.Private.DataContractSerialization/src/System/Runtime/Serialization/ExtensionDataObject.cs

```diff
@@ -13,15 +13,15 @@ public sealed class ExtensionDataObject
     {
         private IList<ExtensionDataMember> _members;
 
-#if USE_REFEMIT || uapaot
```

Comment:
`USE_REFEMIT`  does not seem to be defined anywhere. Maybe another opportunity for cleanup.

Reply:
Won't be use that feature anymore?

Reply:
The name is confusing, I initially thought this is about the entire RefEmit feature set.

Reply:
I do not know what it is used for. I would deal with it in separate change, with input from the owners of data contract serializers.

Reply:
I have the changes in a stash and will send a PR after this one is merged to avoid conflicts.